ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -82,8 +82,8 @@ Create Workbench
         IF    ${pv_existent} == ${TRUE}
             # Use the `Jump to section` links in the page to scroll the section into view
             Click Element    //a[@href="#cluster-storage"]
-            Click Element    xpath=//input[@name="persistent-existing-storage-type-radio"]
-            Select An Existent PV   name=${pv_name}
+            Click Element    //*[@data-testid="existing-storage-button"]
+            Select An Existent PV   ${pv_name}   ods-ci-test-path
```

Comment:
What is `ods-ci-test-path` ?
is it an existing path that was previously created in the PV ?
Can it be variable instead of hardcoded value ?

Reply:
originally the storage mounted to / by default, now a user needs to create a path. Updated and changed to a variable